src/wx/autoupdater/wxmsw/winsparkle-wrapper.cpp

```diff
@@ -46,11 +48,14 @@ WinSparkleDllWrapper::WinSparkleDllWrapper()
     winsparkle_cleanup              = reinterpret_cast<func_win_sparkle_cleanup>(winsparkle_dll->GetSymbol("win_sparkle_cleanup"));
 }
 
+
 WinSparkleDllWrapper::~WinSparkleDllWrapper()
 {
     delete winsparkle_dll;
 
     // Wait for the program to exit and release the DLL before deleting the temp file, otherwise access is denied.
+    // This is needed due to `win_sparkle_init` locking the file until app execution is over.
+    // `win_sparkle_cleanup` does not release the lock.
```

Comment:
Interesting, I didn't know any of this.

I actually submitted a question issue for winsparkle at the time and the maintainer was extremely rude to me, he even deleted the issue. But I fixed the problem anyway.